example-new-architecture/ios/DdSdkReactNativeExample.xcodeproj/project.pbxproj

```diff
@@ -592,14 +592,24 @@
 				);
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				OTHER_CFLAGS = (
+					"$(inherited)",
+					"-DRN_FABRIC_ENABLED",
+				);
 				OTHER_CPLUSPLUSFLAGS = (
 					"$(OTHER_CFLAGS)",
 					"-DFOLLY_NO_CONFIG",
 					"-DFOLLY_MOBILE=1",
 					"-DFOLLY_USE_LIBCPP=1",
+					"-DRN_FABRIC_ENABLED",
```

Comment:
Did you intend to add this define for this PR as well?

Reply:
🤔 Indeed let me check if they're really needed

Reply:
Indeed I don't know how they're added, we can remove them